android/core/src/main/kotlin/com/chicagoroboto/features/sessions/SessionListPresenter.kt

```diff
@@ -42,4 +47,11 @@ class SessionListPresenter(private val sessionProvider: SessionProvider,
             favoriteProvider.removeFavoriteListener(it)
         }
     }
+
+    private fun findCurrentSessionIndex(sessions: List<Session>): Int {
+        // find the current session index
+        val now = Date()
+        val index = sessions.indexOfFirst { now.after(it.startTime) && now.before(it.endTime) }
```

Comment:
Default value here is -1, is that handled? (I.e. before or after the conference)